### PR TITLE
TILA-1164 | Change SpaceType surfaceArea to be float

### PR DESCRIPTION
**Pull request by nicobav**
TILA-1164

### CODE REVIEW

File: api/graphql/tests/test_spaces.py [hunk 1]
```diff
@@ -219,3 +232,34 @@ def test_regular_user_cannot_update(self):
         assert_that(content.get("errors")).is_not_none()
         self.space.refresh_from_db()
         assert_that(self.space.name_fi).is_equal_to("Space1")
+
+
+class SpacesQueryTestCase(GrapheneTestCaseBase, snapshottest.TestCase):
+    @classmethod
+    def setUpTestData(cls):
+        super().setUpTestData()
+        cls.space = SpaceFactory(surface_area=40.5)
```
Comment (jmp) [inline api/graphql/tests/test_spaces.py:241]: Let's add some constant `name` for the space so that it will stay the same through separate test runs. 🙂

**Review by jmp — CHANGES_REQUESTED** [review]

File: api/graphql/tests/test_spaces.py (hunk 1 above)
Comment (nicobav) [reply api/graphql/tests/test_spaces.py]: that's fixed now

File: api/graphql/tests/test_spaces.py [hunk 2]
```diff
@@ -219,3 +232,34 @@ def test_regular_user_cannot_update(self):
...
+            """
+            query {
+                spaces {
+                    edges {
+                        node {
+                            nameFi
+                            surfaceArea
+                            code
+                            maxPersons
+                            code
```
Comment (jmp) [inline api/graphql/tests/test_spaces.py:256]: We can probably remove this extra `code`. 🙂

**Review by jmp — APPROVED** [review]
Looks good, just one minor comment.

**Pull request merged**